validation/src/test/scala/hmda/validation/rules/lar/validity/V215Spec.scala

```diff
@@ -0,0 +1,37 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V215Spec extends LarEditCheckSpec {
+  property("Passes if application date is NA") {
+    forAll(larGen) { lar =>
+      val naLoan = lar.loan.copy(applicationDate = "NA")
+      val newLar = lar.copy(loan = naLoan)
+      newLar.mustPass
+    }
+  }
+
+  property("Passes if actionTakenType is not 6") {
+    forAll(larGen) { lar =>
+      whenever(lar.actionTakenType != 6) {
+        lar.mustPass
+      }
+    }
+  }
+
+  val invalidDate: Gen[String] = Gen.alphaStr.filter(_ != "NA")
```

Comment:
I don't understand why this edit should fail if the date format is bad--that isn't one of the requirements of this edit.